### ISSUE TITLE
clarify relationship between task and request resources

### ISSUE THREAD

**Issue by joostholslag** [SanteonNL/shared-care-planning#45]
creation of a task from an edit/create of a request resource is the most sensible pattern, now it seems to be the revers. Maybe there are good reasons for this pattern?

**Comment by jorritspee** [comment]
A good starting point for clarification is the Request Resource Pattern as defined by HL7: https://www.hl7.org/fhir/workflow.html#request

A couple of lines from that paragraph:
- "Requests are resources that represent the proposal, plan or order for an activity to occur."
- "All requests with an intent of "order" authorize something."
- "the existence of a "Request" instance doesn't necessarily imply that fulfillment will be requested immediately - or even ever."

All of these apply more (solely) to the Request-resource, and not (not so much) to the Task resource.

The common pattern is:
1. Requester wants something to be done for a specific patient (e.g. telemonitoring)
2. This Request is represented in a Request-resource
3. The Requester will then ask people and/or organizations to do this
4. This asking is represented in a Task-resource, that points (via Task.focus) to the Request-resource

Do you agree with this answer @bramwesselo?

**Comment by bramwesselo** [comment]
Yes; the (Service-)request is used to determine what should/could be done. The Task is used to manage the workflow-state of the request (or part of the request) and assign it to an owner (or 'filler').
